kubernetes/agent-debug-events.yaml

```diff
@@ -33,5 +33,6 @@ items:
               imagePullPolicy: IfNotPresent
               args:
               - -feature.install-agents=false
+              - -feature.kubernetes-events=true
```

Comment:
We are enabling by default?

Reply:
This isn't a manifest used anywhere, just a convenience one that developers can use to start an agent that will only listen to k8s event for debugging purposes.